lib/iex/lib/iex/helpers.ex

```diff
@@ -491,6 +491,7 @@ defmodule IEx.Helpers do
   defp all_implemented_protocols_for_term(term) do
     :code.get_path()
     |> Protocol.extract_protocols()
+    |> Enum.sort()
     |> Enum.uniq()
```

Comment:
We can use `:lists.usort` here too, which both sorts and removes duplicates. If we want to keep it this way, then I think the fastest thing to do here is to sort after we reject the protocols, because that would be less data to sort. :)

Reply:
> We can use `:lists.usort` here too, which both sorts and removes duplicates
I am trying to get rid of :lists calls as much possible as you can see in my other PR

Reply:
`:get_path` is misleading. It actually returns a list of paths.